providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/model/FlagParser.java

```diff
@@ -29,10 +29,7 @@ public class FlagParser {
     private static final String FLAG_KEY = "flags";
     private static final String EVALUATOR_KEY = "$evaluators";
     private static final String REPLACER_FORMAT = "\"\\$ref\":(\\s)*\"%s\"";
-
     private static final ObjectMapper MAPPER = new ObjectMapper();
-    private static final Map<String, Pattern> PATTERN_MAP = new HashMap<>();
```

Comment:
This field presents a concurrency issue, since multiple threads can possible parse at the same time, and `computeIfAbsent` can throw an concurrent-modification-exception in Java 9+